### PR TITLE
CI-5054 adding docs around the new AEM servlet health checks

### PR DESCRIPTION
**Pull request by francoisledroff**
## Description
CI-5054 adding docs around the new AEM servlet health checks

hiding the aem 6.3 docs (as we stopped supporting 6.3)

## Note
PR raised against which branch -
 - [x] master (Production)
 - [ ] stage (Stage)

### CODE REVIEW

File: aem/aem_6_3_on_premise_install_6.3.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ This documentation below is specific to `AEM on premise` version 6.3.x (and 6.2.
 4. Configure Adobe I/O Credentials
    1. [configure the Adobe I/O Events user keystore in AEM](aem_keystore_setup.md) 
    2. [configure your workspace in the `Adobe Developer Console`](aem_console_setup.md)
-   3. [add a Cloud Services configuration in AEM](aem_cloud_service_config.md)
+   3. [add a Cloud Services configuration in AEM](aem_6_3_cloud_service_config.md)
```
Comment (shikhartanwar) [inline aem/aem_6_3_on_premise_install_6.3.md:16]: Can we please standardize the naming convention by appending the AEM version as the suffix?
`aem_cloud_service_config_6_3`. 
Right now its a mix of either prefix or suffix in our file names.

File: aem/aem_healthcheck_servlet.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+   
+  
+## Trigger Adobe I/O Events health checks
+
+Once all your configurations are completed, in order to verify **and enable**
```
Comment (shikhartanwar) [inline aem/aem_healthcheck_servlet.md:5]: Why is this required for **enabling** the emission of events?

File: aem/aem_healthcheck_servlet.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+###  AEM Adobe I/O Events Configuration checks
+
+This section shows your AEM identification configurations: 
+* the Link Externalizer name: by default set to `author`
+* the Externalizer author Root Url
+* the Adobe I/O Event Provider instance_id: derived from the above
+* the Adobe I/O Event Provider metadata: by default set to `aem`
+
+### Adobe IO Workspace Configuration checks
```
Comment (shikhartanwar) [inline aem/aem_healthcheck_servlet.md:21]: Let's change `IO` to `I/O` throughout consistently.
```suggestion
### Adobe I/O Workspace Configuration checks
```

File: aem/aem_healthcheck_servlet.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+   ![Health check Servlet output sample](../img/events_aem_healthcheck_servlet.png "Health check for conf-events")
+
+It exposes the following sections:
+
+###  AEM Adobe I/O Events Configuration checks
+
+This section shows your AEM identification configurations: 
+* the Link Externalizer name: by default set to `author`
+* the Externalizer author Root Url
```
Comment (shikhartanwar) [inline aem/aem_healthcheck_servlet.md:17]:
```suggestion
* the Link Externalizer author Root Url
```

File: aem/aem_healthcheck_servlet.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+* the Adobe I/O Event Provider instance_id: derived from the above
+* the Adobe I/O Event Provider metadata: by default set to `aem`
+
+### Adobe IO Workspace Configuration checks
+
+This section shows
```
Comment (shikhartanwar) [inline aem/aem_healthcheck_servlet.md:23]: _Nitpick_
```suggestion
This section shows:
```

File: aem/aem_healthcheck_servlet.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+* the Link Externalizer name: by default set to `author`
+* the Externalizer author Root Url
+* the Adobe I/O Event Provider instance_id: derived from the above
+* the Adobe I/O Event Provider metadata: by default set to `aem`
+
+### Adobe IO Workspace Configuration checks
+
+This section shows 
+* your [`Adobe Developer Console` worskpace OSGI configuration](./aem_console_setup.md): your consumerOrgId, projectId, workspaceId
+* your Project url: the associated https://developer.adobe.com/console deeplink
```
Comment (shikhartanwar) [inline aem/aem_healthcheck_servlet.md:25]: Could you remind me where are we using the project URL in our package? Is this pure metadata?

File: manifest.json [hunk 7]
```diff
@@ -46,12 +46,6 @@
           "path": "adobedocs/adobeio-events/master/using/aem-event-setup.md",
           "title": "AEM Events",
           "pages": [
-            {
-              "importedFileName": "aem_on_premise_install_6.3",
-              "path": "adobedocs/adobeio-events/master/aem/aem_on_premise_install_6.3.md",
-              "title": "'AEM 6.2.x & 6.3.x on Premise' Events",
```
Comment (shikhartanwar) [inline manifest.json:52]: Even though we stopped supporting AEM 6.3, I am not sure if hiding the pages without any prior notice is the way to do it. We should instead mark the 6.3 page as deprecated and remove it only if there are no active integrations. WDYT?

File: aem/aem_6_3_on_premise_install_6.3.md (hunk 1 above)
Comment (francoisledroff) [reply aem/aem_6_3_on_premise_install_6.3.md:16]: good point

File: aem/aem_healthcheck_servlet.md (hunk 2 above)
Comment (francoisledroff) [reply aem/aem_healthcheck_servlet.md:5]: to guarantee the Adobe I/O Events Providers Metadata are persisted in our control plane.

File: aem/aem_healthcheck_servlet.md (hunk 3 above)
Comment (francoisledroff) [reply aem/aem_healthcheck_servlet.md:21]: sure

File: aem/aem_healthcheck_servlet.md (hunk 4 above)
Comment (francoisledroff) [reply aem/aem_healthcheck_servlet.md:17]: sure

File: aem/aem_healthcheck_servlet.md (hunk 5 above)
Comment (francoisledroff) [reply aem/aem_healthcheck_servlet.md:23]: sure

File: aem/aem_healthcheck_servlet.md (hunk 6 above)
Comment (francoisledroff) [reply aem/aem_healthcheck_servlet.md:25]: we are not, this is to help our customer finding their way to the workspace associated with this AEM instance

File: manifest.json (hunk 7 above)
Comment (francoisledroff) [reply manifest.json:52]: sure... I added the aem 6.3 link back in the main `aem-event-setup.md` page

Comment (francoisledroff) [reply manifest.json:52]: however to keep the left side menu cleaner, I removed the 6.3 links from there.

File: aem/aem_healthcheck_servlet.md (hunk 6 above)
Comment (shikhartanwar) [reply aem/aem_healthcheck_servlet.md:25]: oh ok. So we construct this during the health check?

File: manifest.json (hunk 7 above)
Comment (shikhartanwar) [reply manifest.json:52]: Makes sense.

File: aem/aem_healthcheck_servlet.md (hunk 2 above)
Comment (shikhartanwar) [reply aem/aem_healthcheck_servlet.md:5]: Yes but even without the health check, the provider and corresponding event metadata entries will be persisted, right? My question/doubt that comes out of this line is whether running the health check a required step now?

Comment (francoisledroff) [reply aem/aem_healthcheck_servlet.md:5]: yes It is required.

Comment (francoisledroff) [reply aem/aem_healthcheck_servlet.md:5]: if the customers follows the installation step in the exact order, the provider metadata will already be persisted... but better to be safe than sorry...

File: aem/aem_healthcheck_servlet.md (hunk 6 above)
Comment (francoisledroff) [reply aem/aem_healthcheck_servlet.md:25]: yes

File: aem/aem_healthcheck_servlet.md (hunk 2 above)
Comment (shikhartanwar) [reply aem/aem_healthcheck_servlet.md:5]: Ok

**Review by shikhartanwar — APPROVED** [review]
